src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BookList from './components/BookList';
+import BooksForm from './components/BooksForm';
+// import logo from './logo.svg';
+import './App.css';
+
+const App = () => (
+  <div className="App">
+    <BookList />
+    <BooksForm />
+  </div>
+);
+
+export default App;
```

Comment:
pls relocate this file to `src/components` folder

Reply:
Will be resolved in Milestone 3 to avoid misalignment of code